nevergrad/functions/ml/test_mlfunctionlib.py

```diff
@@ -61,9 +61,12 @@
 def test_mltuning_values(cls_params: tp.Dict[str, tp.Any], func_params: tp.Dict[str, tp.Any], expected: float) -> None:
     np.random.seed(12)
     func = mlfunctionlib.MLTuning(**cls_params)
+    if any(params.get("regressor", "") == "mlp" for params in (func_params, cls_params)):
+        # make it way faster
+        func.num_data = 12
+        func._cross_val_num = 2
```

Comment:
This let us run the tests in a few seconds instead of 1min30

Reply:
Just great, thx.